### PR TITLE
Fix incorrect property listing in skipper manifest

### PR DESCRIPTION
**Pull request by sabbyanandan**
Resolves spring-cloud/spring-cloud-dataflow-server-cloudfoundry#441.

### CODE REVIEW

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -108,12 +108,12 @@ env:
...
     SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_CONNECTION_SPACE: {space}
-    SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_CONNECTION_DOMAIN: cfapps.io
     SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_CONNECTION_USERNAME: {email}
     SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_CONNECTION_PASSWORD: {password}
-    SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_DEPLOYMENT_SERVICES: {middlewareServiceName}
     SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_CONNECTION_SKIP_SSL_VALIDATION: false
-    SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_CONNECTION_STREAM_ENABLE_RANDOM_APP_NAME_PREFIX: false
+    SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_DEPLOYMENT_DOMAIN: cfapps.io
+    SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_DEPLOYMENT_SERVICES: {middlewareServiceName}
+    SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_DEPLOYMENT_STREAM_ENABLE_RANDOM_APP_NAME_PREFIX: false
```
Comment (jvalkeal) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: This `STREAM` is extra. Think format is `SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[pws]_DEPLOYMENT_ENABLE_RANDOM_APP_NAME_PREFIX: false`, right?

**Comment by sabbyanandan** [comment]
@jvalkeal: We do one more level of prefix manipulation in CF-server directly - see [here](https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry/blob/master/spring-cloud-dataflow-server-cloudfoundry-autoconfig/src/main/java/org/springframework/cloud/dataflow/server/cloudfoundry/config/CloudFoundryDataFlowServerConfiguration.java#L51). Since Skipper is only for streams, it'd be better to stick with the stream specific property.

**Comment by jvalkeal** [comment]
ok, I was scratching my head from where it might come. good, gonna merge this then.

**Comment by jvalkeal** [comment]
Merged per 7cfb7ab8f6cbc5ca27f7f881f7bd267ef4c3c542

**Pull request closed without merging**